metsim/metsim.py

```diff
@@ -247,6 +249,15 @@ def met_data(self):
             self._met_data['elev'] = self.domain['elev']
             self._met_data['lat'] = self.domain['lat']
             self._met_data['lon'] = self.domain['lon']
+
+            # process constant_vars
+            template_var = list(self.params.get('forcing_vars', None).values())[0]
+            constant_vars = self.params.get('constant_vars', None)
+            for var in constant_vars.keys():
+                constant_value = float(constant_vars[var])
+                self._met_data[var] = self._met_data[template_var].copy()
+                self._met_data[var] += -self._met_data[var] + constant_value
```

Comment:
Clunky - there must be a better way to set a `DataArray` to a constant value.

Reply:
I think you want `xarray.full_like`: http://xarray.pydata.org/en/stable/generated/xarray.full_like.html#xarray.full_like